pulsar-functions/instance/src/main/python/contextimpl.py

```diff
@@ -98,7 +98,7 @@ def record_metric(self, metric_name, metric_value):
       self.accumulated_metrics[metric_name] = AccumulatedMetricDatum()
```

Comment:
we need to also change the context.py to reflect this

Reply:
Fixed